### DISCUSSION TITLE
How to delete the text behind each validation error msg?

### DISCUSSION THREAD

**Discussion by durianwaffle** [eclipse-langium/langium#1713] [Q&A] [answered]
I find that there is a text, which is grammar name at the end of each validation error msg.
Is there any way to delete it?
![image](https://github.com/user-attachments/assets/ce67a2b5-7f73-4b8a-8477-3f729d6b242c)

**Comment by msujew** [comment, marked as answer]
Hey @durianwaffle,

you can override the `DocumentValidator#getSource` method to return an empty string to remove/change the text:

https://github.com/eclipse-langium/langium/blob/2e397013eed1e89ef54d6fb1803d5cf86e0cd5b4/packages/langium/src/validation/document-validator.ts#L197-L213

**Reply by durianwaffle** [reply]
Thank you @msujew !
